shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -67,7 +67,7 @@ class FlutterTizenEngine : public TizenRenderer::Delegate {
                           bool focusable);
 
   // Starts running the engine.
-  bool RunEngine();
+  bool RunEngine(const char* entrypoint);
```

Comment:
Could you update the comment so that it matches the Windows engine's method description?
```cpp
  // Starts running the engine with the given entrypoint. If null, defaults to
  // main().
  //
  // Returns false if the engine couldn't be started.
```

Reply:
Also you need to update `flutter_tizen_engine_unittest.cc`. The unit tests are not currently built by the CI by default. I'm going to enable the tests soon.

Reply:
@xiaowei-guan Please check out the above comment.

Reply:
Ok,I will update it. and how to run unittest?

Reply:
Currently you need a Linux desktop with EFL installed: https://github.com/flutter-tizen/engine/pull/125

You don't have to run the tests if you don't want to set up EFL on desktop.